### PR TITLE
remove limit for low keys

### PR DESCRIPTION
**Pull request by mesmerx**
this pr try to remove the limit for low keys

### CODE REVIEW

**Comment by sybrenstuvel** [comment]
What would this be useful for?

**Comment by mesmerx** [comment]
this help with keys that is less than required key, this seems a little odd but i have one case that i cannot use python-rsa because of that and pycrypto works

**Comment by sybrenstuvel** [comment]
Let me rephrase: which key lengths have a practical use, and still aren't allowed by this library?

**Comment by sybrenstuvel** [comment]
Can you give a concrete example of how you'd use such small keys?

**Comment by sybrenstuvel** [comment]
Closing because of inactivity.
